src/PotentialTypes/EmpiricalPotentials/force.jl

```diff
@@ -1,18 +1,16 @@
 function force(A::AbstractSystem, p::EmpiricalPotential)
-    number_of_particles = length(A.particles)
-    f = [zeros(3) for i = 1:number_of_particles]
+    f = fill(zeros(3), length(A))
     nnlist = neighborlist(A, p.rcutoff)
 
-    for i in nnlist.i
-        j = nnlist.j[i[1]]
-        r = nnlist.r[i[1]]
-        R = nnlist.R[i[1]]
+    for ii in 1:length(A)
```

Comment:
This code change might not actually be correct. The length of nnlist.i can certainly be less than length(A) if there are some atoms that are not neighbors of atom i.